extra-dependencies/xls/build.gradle

```diff
@@ -3,8 +3,8 @@ plugins {
 }
 
 java {
-    sourceCompatibility = JavaVersion.VERSION_11
-    targetCompatibility = JavaVersion.VERSION_11
+    sourceCompatibility = JavaVersion.VERSION_17
+    targetCompatibility = JavaVersion.VERSION_17
```

Comment:
We need to update this in the other maintained branches

Reply:
No I don't think so, as Neo 4.x is supposed to be run with Java 11?